go.mod

```diff
@@ -36,6 +36,7 @@ require (
 	kubevirt.io/api v1.0.0
 	kubevirt.io/containerized-data-importer-api v1.57.0-rc1
 	kubevirt.io/controller-lifecycle-operator-sdk/api v0.2.4
+	kubevirt.io/managed-tenant-quota-api v1.1.0
```

Comment:
actually we didn't create api repo yet please change to:

`kubevirt.io/managed-tenant-quota v1.1.0`